daprdocs/content/en/operations/hosting/kubernetes/kubernetes-dapr-shared.md

```diff
@@ -0,0 +1,84 @@
+---
+type: docs
+title: "Deploy Dapr per-node or per-cluster with Dapr Shared"
+linkTitle: "Dapr Shared"
+weight: 50000
+description: "Learn more about using Dapr Shared as an alternative deployment to sidecars"
+
+---
+
+Dapr automatically injects a sidecar to enable the Dapr APIs for your applications for the best availability and reliability. 
+
+Dapr Shared enables two alternative deployment strategies to create Dapr applications using a Kubernetes `Daemonset`for a per-node deployment or a`Deployment` for a per-cluster deployment. 
+
+- **`DaemonSet`:** When specifying to run Dapr as a Kubernetes `DaemonSet` resource, the daprd container runs on each Kubernetes node. This can reduce network hops between the applications and Dapr. 
+- **`Deployment`:** When running Dapr Shared as a Kubernetes `Deployment`, the Kubernetes scheduler decides in which node the Dapr Shared instance will run.
+
+{{% alert title="Dapr Shared deployments" color="primary" %}}
+For each Dapr application, you need to deploy the Dapr Shared chart using different `shared.appId`s.
+{{% /alert %}}
+
+
+
+## Why Dapr Shared?
+
+By default, when Dapr is installed into a Kubernetes cluster, the Dapr control plane injects a Dapr as sidecar to applications annotated with Dapr annotations ( `dapr.io/enabled: "true"`). Sidecars have many advantages including improved resiliency since there is an instance per application and all communication between the application and the sidecar happens without involving the network.
```

Comment:
```suggestion
By default, when Dapr is installed into a Kubernetes cluster, the Dapr control plane injects Dapr as a sidecar to applications annotated with Dapr annotations ( `dapr.io/enabled: "true"`). Sidecars offer many advantages, including improved resiliency, since there is an instance per application and all communication between the application and the sidecar happens without involving the network.
```